### ISSUE TITLE
IDE0037 codefix introduces extra space, breaks indentation alignment

### ISSUE THREAD

**Issue by binki** [dotnet/roslyn#23729]
**Version Used**: 2.6.0.62329 (5429b35d) (Visual Studio 15.5.1 Community)

**Steps to Reproduce**:

1. Write an expression which triggers IDE0037. For example:
   ```csharp
   public class Class1
   {
       public void M()
       {
           var x = 23;
           Equals(null, new
           {
               x = x,
               y = 23,
           });
       }
   }
   ```
2. Execute the code fix

**Expected Behavior**:

The string of whitespace characters between the opening `{` and the first non-whitespace character (i.e., the identifier itself) remains the same as before. This would preserve the “normal” indent that existed prior to executing the code fix and avoids changing the alignment of the variable wrt neighbors.

```csharp
public class Class1
{
    public void M()
    {
        var x = 23;
        Equals(null, new
        {
            x,
            y = 23,
        });
    }
}
```

**Actual Behavior**:

There is an additional ` ` (space, I haven’t verified, it looks like space) character between the opening `{` and the first non-whitespace character after the `{`. The identifier is now out of alignment with neighbors.

```csharp
public class Class1
{
    public void M()
    {
        var x = 23;
        Equals(null, new
        {
             x,
            y = 23,
        });
    }
}
```

**Comment by jcouv** [comment]
I think this is an issue I fixed in 15.6. (PR https://github.com/dotnet/roslyn/pull/22797)

Here's the original issue: https://github.com/dotnet/roslyn/issues/22347

@binki I'll go ahead and close. Feel free to re-open if you still notice a problem on 15.6 preview 1 (or later).

**Comment by binki** [comment]
It appears to be fixed in 15.6.0 Preview 1.0. Now I know how to install and use that ;-). Thanks!
